ddl/partition.go

```diff
@@ -92,6 +92,7 @@ func buildHashPartitionDefinitions(ctx sessionctx.Context, d *ddl, s *ast.Create
 			return errors.Trace(err)
 		}
 		defs[i].ID = pid
+		defs[i].Name = model.NewCIStr(fmt.Sprintf("p%v", i))
```

Comment:
s/ fmt.Sprintf("p%v", i) / fmt.Sprintf("%v", i)? `Hash Partition` does not care about partition names.

Reply:
@ciscoxll select partition need p0,p1... name in hash partition table.
In mysql:
```sql
mysql > create table th (a int) partition by hash (a) partitions 5;
Query OK, 0 rows affected
Time: 0.014s
mysql > select * from th partition (p0)
+---+
| a |
+---+
```

Reply:
Ok, I see.